src/components/monitorLog/HumanActivity.jsx

```diff
@@ -17,32 +17,18 @@ import {
   Tooltip,
 } from '@chakra-ui/react';
 
-function PredatorsHumanActivity() {
+function HumanActivity() {
```

Comment:
All human activity fields need to be stored somewhere. Implement this logic with useState hooks